tasks/Windows.yml

```diff
@@ -21,6 +21,46 @@
   when:
     - ansible_architecture == "64-bit"
 
+- name: "Windows | Check if Zabbix agent is present"
+  win_stat:
+    path: '{{ zabbix_win_exe_path }}'
+  register: agent_file_info
+
+- name: "Windows | Get Installed Zabbix Agent Version"
+  win_file_version:
+    path: "{{ zabbix_win_exe_path }}"
+  register: zabbix_win_exe_info
+  when: agent_file_info.stat.exists
+
+- name: Windows | Checking Update
```

Comment:
Just my "autism" here, but could you add double quotes in the name like any other name in the playbook? 😄

Reply:
Sure no problem, I missed that one :-)